backend/src/examples/flask_examples/sample_queries.py

```diff
@@ -0,0 +1,14 @@
+from enum import Enum
+
+
+class ConnectionQuerySamples(Enum):
```

Comment:
this is definitely not an enum

Reply:
I know it's kind of overusage of enum, but it's usefull for testing as Pycharm autocomplete name of variable for you, and dict doesn't.

Reply:
It would be much simpler to have all examples defined in the `example_connection.py` file.